### PR TITLE
Changes to README.md only

### PR DESCRIPTION
**Pull request by ljhwang**
Contains suggested changes to the README.md files only.  

The original files mixed instructions with examples. These updates includes for each section instructions in plain text code blocks and example text.

An additional section was added to more closely align with the Contributing Software Checklist.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,32 +11,45 @@ Link to any figshare resources as follows: [![pdf manual](https://img.shields.io
...
+New projects should begin by placing a copy of this repository into your project repository and modifying the contents of each file as appropriate.
 
 
 Citing SOFTWARE_TEMPLATE
 ------------------------
 
-Include information about how to cite SOFTWARE_TEMPLATE, e.g. using https://geodynamics.org/cig/abc. We also recomment adding a codemeta file, e.g. generated using https://codemeta.github.io/codemeta-generator/ to make your software citable.
+`Include information about how to cite SOFTWARE_TEMPLATE. Consider adding citations to both publications and the software itself. For examples, see:` [https://geodynamics.org](https://geodynamics.org/cig/abc).
 
+`We also recommend adding a codemeta file (`codemeta.json`). Generate this file to make your code citable:`[https://codemeta.github.io/codemeta-generator/]( https://codemeta.github.io/codemeta-generator/ ).
```
Comment (gassmoeller) [inline README.md]: Maybe make that a link to the file in the repo?

```suggestion
`We also recommend adding a codemeta file (`[codemeta.json](codemeta.json)`). Generate this file to make your code citable:`[https://codemeta.github.io/codemeta-generator/]( https://codemeta.github.io/codemeta-generator/ ).
```

File: README.md [hunk 2]
```diff
@@ -11,32 +11,45 @@ Link to any figshare resources as follows: [![pdf manual](https://img.shields.io
...
+
+SOFTWARE_TEMPLATE is free to use and does not require citation or acknowledgement.
 
 Contributing to our software
 ----------------------
+`Describe how to contribute to the project.  Consider pointing to a CONTRIBUTING.MD FILE.`
```
Comment (gassmoeller) [inline README.md]:
```suggestion
`Describe how to contribute to the project. Consider pointing to a CONTRIBUTING.MD FILE.`
```

File: README.md [hunk 3]
```diff
@@ -45,22 +58,30 @@ participating! We have collected a set of guidelines and advice on how to get
 involved in the community and keep them in the
 [CONTRIBUTING.md](CONTRIBUTING.md) file in the repository.
 
+Feedback and support
+----------------------
+`Explain how a user should provide feedback, report bugs, and get help.`
+
+For support, see our forum at: https://community.geodynamics.org/.
+
+To contribute to this project, see `CONTRIBUTING.md`.
```
Comment (gassmoeller) [inline README.md]:
```suggestion
To contribute to this project, see [CONTRIBUTING.md](CONTRIBUTING.md).
```

**Review by gassmoeller — COMMENTED** [review]
Thanks, I like it. I made some small suggestions after that I will merge.

File: README.md (hunk 1 above)
Comment (ljhwang) [reply README.md]: ok to merge. Thanks.

File: README.md (hunk 2 above)
Comment (ljhwang) [reply README.md]: ok to merge. Thanks.

File: README.md (hunk 3 above)
Comment (ljhwang) [reply README.md]: ok to merge. Thanks.

**Pull request merged**